Src/java/engine-fhir/src/main/java/org/opencds/cqf/cql/engine/fhir/converter/Dstu2FhirTypeConverter.java

```diff
@@ -227,7 +228,35 @@ public IBase toFhirTuple(Tuple value) {
             return null;
         }
 
-        throw new NotImplementedException("can't convert Tuples");
+        var parameters = new Parameters();
+        if (value.getElements().isEmpty()) {
+            return parameters;
+        }
+
+        var param = parameters.addParameter();
+        for (String key : value.getElements().keySet()) {
+            var part = param.addPart();
+            part.setName(key);
+            var element = value.getElements().get(key);
+            if (element == null) {
+                part.addExtension()
+                        .setUrl(DATA_ABSENT_REASON_EXT_URL)
+                        .setValue(new CodeType(DATA_ABSENT_REASON_UNKNOWN_CODE));
+            } else if (element instanceof Collection && ((Collection<?>) element).isEmpty()) {
+                part.addExtension().setUrl(EMPTY_LIST_EXT_URL).setValue(new BooleanType(true));
```

Comment:
Same issue here, the cqf-isEmptyList extension needs to be on the value, not the part:

```xml
<parameter>
  <extension url="http://hl7.org/fhir/StructureDefinition/cqf-cqlType">
    <valueString value="List<Tuple{id:System.Integer,name:System.String,address:List<Tuple{street:System.String,city:System.String,state:System.String}>}>"/>
  </extension>
  <name value="CQLEmptyListExample"/>
  <valueBoolean>
    <extension url="http://hl7.org/fhir/StructureDefinition/cqf-isEmptyList">
      <valueBoolean value="true"/>
    </extension>
  </valueBoolean>
</parameter>
```

Reply:
Same situation - we don't have that type information. Is a `valueBoolean` the correct value for a List of Tuples?